repository_test.go

```diff
@@ -3333,3 +3333,20 @@ func BenchmarkPlainClone(b *testing.B) {
 		clone(b)
 	}
 }
+
+func (s *RepositorySuite) TestGetNoteNotExisting(c *C) {
+	r, _ := Init(memory.NewStorage(), nil)
+	err := r.clone(context.Background(), &CloneOptions{URL: s.GetBasicLocalRepositoryURL()})
+	c.Assert(err, IsNil)
+
+	head, err := r.Head()
+	c.Assert(err, IsNil)
+
+	commit, err := r.CommitObject(head.Hash())
+	c.Assert(err, IsNil)
+
+	note, err := r.GetNote(commit)
```

Comment:
We need a test case that also includes notes in.

You can create a sample repository with 1-2 commits with some notes. Then you submit its pack as a PR to https://github.com/go-git/go-git-fixtures.

That would enable you to refer to that pack via `ByTag` as we have in other parts of the code base.

Reply:
Now that we got your [changes](https://github.com/go-git/go-git-fixtures/commit/e089218a43607121fbf7600d940ce7b44ce2f2e8) merged in the other repo, we just need to update the go-git-fixtures version to the latest.